ocamlbuild_js_of_ocaml.ml

```diff
@@ -54,31 +54,24 @@ let link_opts prod =
   (* It doesn't do it with 'query' command, we have to it manually. *)
   let cmd = "-format" :: "pkg_%p" :: "-r" :: all_pkgs in
   let predicates_pkgs = ocamlfind cmd (fun ic -> input_line ic) in
-  let all_predicates =
-    String.concat "," (("javascript" :: predicates) @ predicates_pkgs)
+  let all_predicates = String.concat "," (predicates @ predicates_pkgs) in
+  (* query findlib for jsoo runtime files *)
+  let cmd =
+    "-format" :: "%+(jsoo_runtime)" :: "-r" :: "-predicates" :: all_predicates :: all_pkgs
```

Comment:
@gerdstolpmann, I couldn't find documentation about the `%+(NAME)` syntax. Is it documented anywhere ? Is it ok to use it outside of findlib itself ?